integration/mediation-tests/tests-transport/src/test/java/org/wso2/carbon/esb/jms/transport/test/JMSSenderStaleConnectionsTestCase.java

```diff
@@ -101,13 +99,16 @@ public void staleConnectionsTestJMSProxy() throws Exception {
                 break;
             }
         }
-        Assert.assertTrue(!isExceptionThrown, "Sender Side Stale connections handling test passed");
+        Assert.assertFalse(isExceptionThrown, "Sender Side Stale connections handling test failed");
     }
 
     @AfterClass(alwaysRun = true)
     public void destroy() throws Exception {
-        super.cleanup();
-        stopBroker();
+        try {
+            super.cleanup();
+        } finally {
+            stopBroker();
```

Comment:
Will stopping the broker here afftect other test cases since we only start the broker once now?

Reply:
The broker used here is a different one. We're not stopping the commonly started broker.

Reply:
What do you mean by different? Is it with different configurations?

Reply:
Fixed. Edited the axis2 conf to have port 61616 and refactored required artifacts.